src/pages/index.tsx

```diff
@@ -551,9 +559,16 @@ const HomePage: NextPage<HomePageProps> = ({ newsItem }) => {
                 </div>
 
                 <div className="max-w-[34.0625rem] text-black text-xl-rfs font-normal leading-6 tracking-[0.03125rem]">
-                  Our data discovery platform provides access to spatially
-                  indexed and curated databases, specifically designed for
-                  conducting health equity research.
+                  <p>
+                    Looking for community-level SDOH for your project? Explore the
+                    Data Discovery search tool, with or without an AI assist, to
+                    identify high-quality data across the United States.
+                  </p>
+                  <br/>
+                  <p>
+                    Review data availability across topics, various spatial scales
```

Comment:
From Marc:
> Looking for community-level SDOH for your project? Explore the Data Discovery search tool, with or without an AI assist, to identify high-quality data across the United States.
Review data availability across topics, various spatial scales (i.e. census tract vs county), and time periods alongside usage tips.

This was caused by text-wrapping behavior on larger monitors - updating the Data Discovery flavor text here to remove the word "various" should solve the problem, and maintain level weight between the left and right sections here :+1:

```suggestion
                    Review data availability across topics, spatial scales
```